### ISSUE TITLE
[REVIEW]: ForestAtRisk: A Python package for modelling and forecasting deforestation in the tropics

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2975]
**Submitting author:** @ghislainv (<a href="http://orcid.org/0000-0002-1685-4997">Ghislain Vieilledent</a>)
**Repository:** <a href="https://github.com/ghislainv/forestatrisk" target ="_blank">https://github.com/ghislainv/forestatrisk</a>
**Version:** v1.0
**Editor:** @kbarnhart
**Reviewer:** @molgor, @ethanwhite
**Archive:** <a href="https://doi.org/10.5281/zenodo.4570632" target="_blank">10.5281/zenodo.4570632</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/b8fe1d3815c7385a1c3e454270bee041/status.svg)](https://joss.theoj.org/papers/b8fe1d3815c7385a1c3e454270bee041)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/b8fe1d3815c7385a1c3e454270bee041"><img src="https://joss.theoj.org/papers/b8fe1d3815c7385a1c3e454270bee041/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/b8fe1d3815c7385a1c3e454270bee041/status.svg)](https://joss.theoj.org/papers/b8fe1d3815c7385a1c3e454270bee041)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/ghislainv/forestatrisk" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@molgor & @ethanwhite, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @kbarnhart know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ghislainv/forestatrisk">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ghislainv) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ghislainv/forestatrisk">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ghislainv) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @molgor, @ethanwhite it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1038/s41586-020-2649-2 is OK
- 10.1007/bf00116466 is OK
- 10.1007/978-3-319-60801-3_36 is OK
- 10.1016/S0304-3800(02)00059-5 is OK
- 10.1111/j.1461-0248.2004.00702.x is OK
- 10.1038/nature10425 is OK
- 10.1126/science.aam5962 is OK
- 10.1126/science.293.5530.657 is OK
- 10.1016/j.foreco.2015.06.014 is OK
- 10.1101/2020.09.17.295774 is OK
- 10.1111/j.1755-263X.2008.00027.x is OK
- 10.1016/j.envsoft.2013.09.010 is OK
- 10.1038/nature04389 is OK
- 10.5281/zenodo.3553579 is OK
- 10.1201/b10905-5 is OK
- 10.1007/s10980-009-9355-7 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: :point_left:

**Comment by whedon** [comment]
:wave: @molgor, please update us on how your review is going.

**Comment by whedon** [comment]
:wave: @ethanwhite, please update us on how your review is going.

**Comment by kbarnhart** [comment]
@molgor @ethanwhite, we have an automatic reminder two weeks into the review. Feel free to provide an update if you have one.

**Comment by molgor** [comment]
Hi Katy,

I was trying to start the review now. However, it seems that the invitation sent from @weldon has expired. 

Do I need a new one?

Cheers,

Juan

**Comment by molgor** [comment]
I'll try using the command...
# Re-invite a reviewer (if they can't update checklists)
@whedon re-invite @molgor  as reviewer

**Comment by kbarnhart** [comment]
@whedon re-invite @molgor as reviewer

**Comment by whedon** [comment]
OK, the reviewer has been re-invited.

@molgor please accept the invite by clicking this link: https://github.com/openjournals/joss-reviews/invitations

**Comment by kbarnhart** [comment]
@molgor here is the re-invite link.

**Comment by molgor** [comment]
Thank you very much Katy. 

Sorry for answering so late.

Best,

Juan

**Comment by molgor** [comment]
# Checklist for ForestAtRisk repository
## Review checklist for @molgor
### Conflict of interest

- [X] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [X] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [X] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ghislainv/forestatrisk">repository url</a>?
  > Yes, the repository can be downloaded without any problem using:
  `git clone` https://github.com/ghislainv/forestatrisk
- [X] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
  > Yes, the repository includes a file with the GNU - GPLv3 in plain text.
- [X] **Contribution and authorship:** Has the submitting author (@ghislainv) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
  > Yes, according to the Github /insights/ section, the user @ghislainv (i.e. Ghislain Vieilledent ORCID: 0000-0002-1685-4997) is the only contributor and developer of the software,.

- [X] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)
  > Yes, the software satisfies the suggested requirements list:
  - [x] Age of software (the software began in 2017)
  - [x] Total lines of code (LOC) (more than 1000 LOC) 
  - [x] Whether the software has already been cited in academic papers.
   > It is my understanding that they have a pre-print paper citing it.

- [x] Whether the software is sufficiently useful that it is likely to be cited by your peer group. 
  > Certainly, the targeted problem and the statistical methods used are in need by the environmental science community. There are not many Python library capable of implementing CAR processes. The software would allow further developments, analysis and software ensembles.

### Functionality

- [X] **Installation:** Does installation proceed as outlined in the documentation?
  > Yes, the instruction given for the conda package manager worked 100% for installing all dependencies and the library itself, However, I did not check the installation with `virtualenv` The installation test performed without any problem using the
  `$forestatrisk` command (as stated in the README.md file).
- [x] **Functionality:** Have the functional claims of the software been confirmed?
 >  Yes, I was able to perform all the processes showed in the Getting Started tutorial. All of them explained in the documentation and manuscript.
- [X] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)
  > I do not fully undestand this. The software was able to run successfully and without a high impact on my laptop computer. Bayesian approaches (CAR models in particular) are computationally intensive, the software seems to perform well using retail hardware.

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
  *minor Issue*
  > Yes, this statement is clear, as well as their target audience and applicability. However, this information is only present in the paper section and not in the general documentation. It would be valuable to include this section, either in the main repository (i.e. README.md) or in the full documentation website.
- [X] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
  > yes, instructions brief and clear. The installation is automated and do not require to install dependencies manually.
- [X] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
  > Yes, however the documentation is not easily accessible within the main repository. I recommend to add links to the Jupyter notebook and the documentation in the README.md. A section or subsection would be enough.
- [X] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
  > Yes, complete documentation for all visible functions. The author used best practices for documenting the software. Specifically, the use of Sphinx for extracting and building the web documentation.
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
  *Issue*
  > Partial, some automated tests are implemented in folder `\test`. However not all of these tests passed. I am appending the results here.

  `======================================== short test summary info ========================================
FAILED test/test_get_started.py::test_model_binomial_iCAR - assert False
FAILED test/test_get_started.py::test_rho - assert False
=============================== 2 failed, 15 passed, 2 warnings in 51.58s ===============================
:100.0%, mean accept. rates= beta:0.350, rho:0.443`

> I'll report this in the ´issues´ section of the repository. 

- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
  *Issue*
  > No, links for /Community guidelines/ and a /Code of Conduct/ are mentioned in the last section of the README.md file. However, these links are broken. Please amend this issue.

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
  > Yes
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
  > Yes
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
  > Yes, the author provides a comprehensive list of environmental software for modelling vegetation covers. The author clearly states the limitations of that software and how the proposed project contributes to the field with an innovative, flexible and open source tool. 
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
  > The writing style is within expected academic quality. There were some typos that I suggest in a PR, however, it is not my intention to force the author to acknowledge my suggestions as contributions to the software and they may be dismissed if the author considers so.
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?
  > Yes, all references are appropriately cited and in natbib format (I believe). Also, most of them include their corresponding DOI links.

**Comment by kbarnhart** [comment]
@molgor thanks for contributing your review. Are you still not able to edit the check boxes above?

@ethanwhite a friendly reminder that we are about four weeks into the requested six week review time period. 

As always, let me know if you have any questions.

**Comment by ethanwhite** [comment]
Thanks for the reminder @kbarnhart! I've now completed my review and checked everything excepted `Statement of need` in the `Documentation` section. As @molgor noted this is well covered in the paper but generally absent from the `README` and website. I think a few sentences on both should take care of this.

I checked two boxes that @molgor didn't so I'll briefly note my thinking:
* I considered `Automated tests` satisfied because they are running successfully on GitHub Actions: https://github.com/ghislainv/forestatrisk/actions
* I considered `Community guidelines` satisfied because they are present in the standard named files in the standard location so I knew where to find them even though the links in the README were broken. I submitted a PR to fix the links: https://github.com/ghislainv/forestatrisk/pull/37

@ghislainv - Really nice work! An important package that's been really well developed and documented. It's a pleasure to review good software.

**Comment by molgor** [comment]
Hi @kbarnhart, 
I agree.  I'm happy to mark the missing items if @ghislainv accepts @ethanwhite 's PR.

Congratulations @ghislainv, it is a really neat work!

**Comment by ghislainv** [comment]
Thanks a lot @molgor and @ethanwhite for reviewing the software and for your positive feedbacks. Thanks @kbarnhart for handling the paper review. I really appreciate. I will make the necessary changes to the software documentation and paper and will keep you informed soon.

**Comment by ghislainv** [comment]
@kbarnhart, @molgor, @ethanwhite: I have answered to your remarks and taken into account your suggestions. In particular, I have added a "Statement of Need" section in the README file of the software and in the Home page of the website.

**Comment by ghislainv** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: :point_left:

**Comment by kbarnhart** [comment]
@ghislainv thanks for making changes in response to the reviewer comments. I've now linked all the issues and PRs associated with this review. 

@ethanwhite and @molgor thank you for contributing reviews. If @ghislainv has addressed your concerns could you please ensure you have checked off all of the checkboxes and affirm in a comment here that you recommend this submission for acceptance.

**Comment by ethanwhite** [comment]
Looks great! I recommend this submission for acceptance.

**Comment by molgor** [comment]
Agree! I also recommend it for acceptance. 
🎉

**Comment by kbarnhart** [comment]
@ethanwhite and @molgor thank you for your reviews and confirming that you recommend this be accepted.

@ghislainv at this stage I will do some final checks (e.g., paper proofreading) and will then ask you to do a few steps before this can move to the final stages of publication. I anticipate finishing my steps in the next few days.

**Comment by kbarnhart** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1038/s41586-020-2649-2 is OK
- 10.1007/bf00116466 is OK
- 10.1007/978-3-319-60801-3_36 is OK
- 10.1016/S0304-3800(02)00059-5 is OK
- 10.1111/j.1461-0248.2004.00702.x is OK
- 10.1038/nature10425 is OK
- 10.1126/science.aam5962 is OK
- 10.1126/science.293.5530.657 is OK
- 10.1016/j.foreco.2015.06.014 is OK
- 10.1101/2020.09.17.295774 is OK
- 10.1111/j.1755-263X.2008.00027.x is OK
- 10.1016/j.envsoft.2013.09.010 is OK
- 10.1038/nature04389 is OK
- 10.5281/zenodo.3553579 is OK
- 10.1201/b10905-5 is OK
- 10.1007/s10980-009-9355-7 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by kbarnhart** [comment]
Note: I'm going to check off @molgor 's check boxes as all boxes are checked off in [this comment](https://github.com/openjournals/joss-reviews/issues/2975#issuecomment-780272615)

**Comment by kbarnhart** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: :point_left:

**Comment by kbarnhart** [comment]
@ghislainv I've made a PR with some copyedits and a request for minor revision to Figure 1 at  ghislainv/forestatrisk#40

Once that is merged, I'll do a check that the PDF compiles correctly and we can then move to the final stages of the JOSS process.

**Comment by kbarnhart** [comment]
@ghislainv just wanted to check in if you had any questions about my PR.

**Comment by ghislainv** [comment]
@kbarnhart, thanks for proofreading the paper, I will integrate your changes. I am currently modifying the figure of the paper and will keep you informed when it's done.

**Comment by ghislainv** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: :point_left:

**Comment by ghislainv** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: :point_left:

**Comment by ghislainv** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02975/joss.02975/10.21105.joss.02975.pdf) :page_facing_up: :point_left:

**Comment by ghislainv** [comment]
@kbarnhart, I have integrated your suggestions on the text of the paper. I have also modified the figure to include lat/long coordinates and a scale bar.

**Comment by kbarnhart** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1038/s41586-020-2649-2 is OK
- 10.1007/bf00116466 is OK
- 10.1007/978-3-319-60801-3_36 is OK
- 10.1016/S0304-3800(02)00059-5 is OK
- 10.1111/j.1461-0248.2004.00702.x is OK
- 10.1038/nature10425 is OK
- 10.1126/science.aam5962 is OK
- 10.1126/science.293.5530.657 is OK
- 10.1016/j.foreco.2015.06.014 is OK
- 10.1101/2020.09.17.295774 is OK
- 10.1111/j.1755-263X.2008.00027.x is OK
- 10.1016/j.envsoft.2013.09.010 is OK
- 10.1038/nature04389 is OK
- 10.5281/zenodo.3553579 is OK
- 10.1201/b10905-5 is OK
- 10.1007/s10980-009-9355-7 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by kbarnhart** [comment]
@ghislainv thanks for making those changes, at this point could you:
- [ ] Make a tagged release of your software, and list the version tag of the archived version here.
- [ ] Archive the reviewed software in Zenodo or a similar service (e.g., figshare, an institutional repository)
- [ ] Check the archival deposit (e.g., in Zenodo) has the correct metadata. This includes the title (should match the paper title) and author list (make sure the list is correct and people who only made a small fix are not on it). You may also add the authors' ORCID.
- [ ] Please list the DOI of the archived version here.

I can then move forward with accepting the submission.

**Comment by ghislainv** [comment]
@kbarnhart, thanks for following up. I have archived version 1.0 of the `forestatrisk` package on Zenodo:
- [x] Make a tagged release of your software, and list the version tag of the archived version here. ➡️ forestatrisk v1.0
- [x] Archive the reviewed software in Zenodo or a similar service (e.g., figshare, an institutional repository) ➡️ archived on Zenodo
- [x] Check the archival deposit (e.g., in Zenodo) has the correct metadata. This includes the title (should match the paper title) and author list (make sure the list is correct and people who only made a small fix are not on it). You may also add the authors' ORCID. ➡️ Metadata (title and authors) are correct.
- [x] Please list the DOI of the archived version here. ➡️ [10.5281/zenodo.4570632](https://doi.org/10.5281/zenodo.4570632)

**Comment by kbarnhart** [comment]
@whedon set version 1.0

**Comment by whedon** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

```
@whedon commands
```

**Comment by kbarnhart** [comment]
@whedon set v1.0 as version

**Comment by whedon** [comment]
OK. v1.0 is the version.

**Comment by kbarnhart** [comment]
@whedon set 10.5281/zenodo.4570632 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.4570632" target="_blank">10.5281/zenodo.4570632</a> is the archive.

**Comment by kbarnhart** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by kbarnhart** [comment]
@ghislainv I've now recommended that this submission be accepted and published the @openjournals/joss-eics will handle final acceptance. Congratulations. 

@molgor and @ethanwhite thank you for contributing reviews to this submission.

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1038/s41586-020-2649-2 is OK
- 10.1007/bf00116466 is OK
- 10.1007/978-3-319-60801-3_36 is OK
- 10.1016/S0304-3800(02)00059-5 is OK
- 10.1111/j.1461-0248.2004.00702.x is OK
- 10.1038/nature10425 is OK
- 10.1126/science.aam5962 is OK
- 10.1126/science.293.5530.657 is OK
- 10.1016/j.foreco.2015.06.014 is OK
- 10.1101/2020.09.17.295774 is OK
- 10.1111/j.1755-263X.2008.00027.x is OK
- 10.1016/j.envsoft.2013.09.010 is OK
- 10.1038/nature04389 is OK
- 10.5281/zenodo.3553579 is OK
- 10.1201/b10905-5 is OK
- 10.1007/s10980-009-9355-7 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2119

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2119, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by danielskatz** [comment]
@ghislainv - this mostly looks good, but I've suggested some changes in https://github.com/ghislainv/forestatrisk/pull/47 - please merge this or let me know what you disagree with.  (Note that the paper had both UK and US spelling - I've chosen US to be consistent, though UK consistently would also be ok.)

**Comment by kbarnhart** [comment]
Thanks @danielskatz!

**Comment by ghislainv** [comment]
@danielskatz - thanks for your edits, I have merged them.
@kbarnhart , @molgor , @ethanwhite - thanks again for the review of the software and paper.

**Comment by danielskatz** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2120

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2120, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1038/s41586-020-2649-2 is OK
- 10.1007/bf00116466 is OK
- 10.1007/978-3-319-60801-3_36 is OK
- 10.1016/S0304-3800(02)00059-5 is OK
- 10.1111/j.1461-0248.2004.00702.x is OK
- 10.1038/nature10425 is OK
- 10.1126/science.aam5962 is OK
- 10.1126/science.293.5530.657 is OK
- 10.1016/j.foreco.2015.06.014 is OK
- 10.1101/2020.09.17.295774 is OK
- 10.1111/j.1755-263X.2008.00027.x is OK
- 10.1016/j.envsoft.2013.09.010 is OK
- 10.1038/nature04389 is OK
- 10.5281/zenodo.3553579 is OK
- 10.1201/b10905-5 is OK
- 10.1007/s10980-009-9355-7 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by danielskatz** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1366743011191832577) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/2121
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02975](https://doi.org/10.21105/joss.02975)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by danielskatz** [comment]
Congratulations to @ghislainv (Ghislain Vieilledent)!!

And thanks to @molgor and @ethanwhite for reviewing, and @kbarnhart for editing!

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02975/status.svg)](https://doi.org/10.21105/joss.02975)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02975">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02975/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02975/status.svg
   :target: https://doi.org/10.21105/joss.02975
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02975/status.svg)](https://doi.org/10.21105/joss.02975)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss

**Comment by paulrougieux** [comment]
Dear @ghislainv , your paper has been marked as a similar JOSS papers, and the process suggested to "click through to the review issue for that paper and invite one or more of the authors". Would you like to review my paper [Cobwood: Enhancing Forest Economics Model Reusability Through Labeled Panel Data Structures](https://github.com/openjournals/joss-reviews/issues/8587#issuecomment-3071248774) ?

**Comment by kyleniemeyer** [comment]
@ghislainv please disregard, that submission does not yet have an editor. Any review requests will come from a JOSS editor only.
